x-pack/plugins/security_solution/public/detections/components/rules/rule_preview/index.tsx

```diff
@@ -304,83 +222,44 @@ const RulePreviewComponent: React.FC<RulePreviewProps> = ({
       )}
       <EuiFormRow
         label={i18n.QUERY_PREVIEW_LABEL}
-        helpText={HelpTextComponent}
         error={undefined}
         isInvalid={false}
         data-test-subj="rule-preview"
         describedByIds={['rule-preview']}
       >
-        <EuiFlexGroup>
-          <EuiFlexItem grow={1}>
-            {showAdvancedOptions ? (
-              <EuiSuperDatePicker
-                start={startDate}
-                end={endDate}
-                onTimeChange={onTimeChange}
-                showUpdateButton={false}
-                isDisabled={isDisabled}
-                commonlyUsedRanges={timeRanges}
-              />
-            ) : (
-              <Select
-                id="preview-time-frame"
-                options={getTimeframeOptions(ruleType)}
-                value={timeFrame}
-                onChange={(e) => setTimeFrame(e.target.value as Unit)}
-                aria-label={i18n.QUERY_PREVIEW_SELECT_ARIA}
-                disabled={isDisabled}
-                data-test-subj="preview-time-frame"
-              />
-            )}
-          </EuiFlexItem>
+        <EuiFlexGroup alignItems="center" responsive={false} gutterSize="s">
+          <EuiSuperDatePicker
+            start={startDate}
+            end={endDate}
+            isDisabled={isDisabled}
+            onTimeChange={onTimeChange}
+            showUpdateButton={false}
+            commonlyUsedRanges={timeRanges}
+            onRefresh={onTimeframeRefresh}
+            data-test-subj="preview-time-frame"
+          />
           <EuiFlexItem grow={false}>
-            <PreviewButton
-              fill
-              isLoading={isPreviewRequestInProgress}
-              isDisabled={isDisabled || !areRelaventMlJobsRunning}
-              onClick={handlePreviewClick}
-              data-test-subj="queryPreviewButton"
-            >
-              {i18n.QUERY_PREVIEW_BUTTON}
-            </PreviewButton>
+            <EuiSuperUpdateButton
+              isDisabled={isDisabled}
```

Comment:
One quick note: If you have an errored date selection (i.e. from: future, to: past) you are still able to refresh the query. It seems to default to whatever the prior working query was, but the global KQL date picker disables refresh on that. You may need to add the logic for that as well here

Reply:
done